### DISCUSSION TITLE
firgate and H.265 SHould it not work on Chrome or Edge?

### DISCUSSION THREAD

**Discussion by MrCaspan** [blakeblackshear/frigate#10776] [General]
Sorry to open yeat another thread on the fact you cannot view H.265 recording on frigate. From what I can tell [Chrome and Edge](https://caniuse.com/hevc) both support H.265 as of many versions ago but I still cannot seem to playback a H.265 recording in frigate. I thought the issue was always that the browser didn't support H.265? So if they support it now why can I not play back my H.265 recordings? 

FYI I have tried on Newest Edge and Chrome! I can download and VLC will play it and it confirms it's H.265

**Comment by NickM-27** [comment]
You can use media-internals in chrome to see what is going wrong. I have all my cameras set to h265 and it works without issues. It can of course depend on your OS (that you are viewing with), some linux OS still don't support it even on chrome.

I believe edge requires a plugin

**Reply by MrCaspan** [reply]
Okay I am using Windows 11 Pro and also Android (latest version on a Pixel Pro). I will test with the plugin and report back!!

**Reply by MrCaspan** [reply]
it seems to be working now. strange as this was not working before but I have rebuilt my laptop since
